<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>재고현황</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>361</width>
            <height>164</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_prod_number">
               <property name="text">
                <string>상품번호</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_number">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_prod_category">
               <property name="text">
                <string>상품 카테고리</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_category">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_prod_name">
               <property name="text">
                <string>상품명</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_name"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_prod_price">
               <property name="text">
                <string>상품가격</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_price"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_prod_adult">
               <property name="text">
                <string>구매연령제한</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_adult"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_prod_amount">
               <property name="text">
                <string>상품수량</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prod_amount">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="btn_search">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>검색</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>0</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>발주</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>520</x>
            <y>0</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>판매</string>
          </property>
         </widget>
         <widget class="QLabel" name="prod_img">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>30</y>
            <width>210</width>
            <height>220</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="teamprod">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>btn_second_to_third()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_second_to_third()</slot>
 </slots>
</ui>
